src/index.ts

```diff
@@ -57,12 +57,11 @@ export function hasOwn<T>(obj: T, key: string) {
  * @param {string} key
  * @returns {unknown}
  */
-export function ownProperty(obj: Record<string, number | RegExp>, key: string) {
-  if (hasOwn(obj, key)) {
-    return obj[key];
-  } else {
-    return undefined;
-  }
+export function ownProperty(
```

Comment:
see https://mdn.io/relect-get

Reply:
> see https://mdn.io/relect-get

I believe the `Reflect.get` static method would also be able to access any inherited properties (e.g., from the base `Object` superclass &mdash; all the properties from the constructor would be included), which i think we'd like to avoid. We would only want to have access to all [enumerable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/propertyIsEnumerable) [own properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) iiuc.